packages/react-variant-view/src/component/headerAnnotation/HeaderAnnotation.tsx

```diff
@@ -363,7 +363,9 @@ export default class HeaderAnnotation extends React.Component<
                 <a
                     href={`https://www.genomenexus.org/annotation/${
                         this.props.variant
-                    }?fields=${ANNOTATION_QUERY_FIELDS.join(',')}`}
+                    }?isoformOverrideSource=mskcc&fields=${ANNOTATION_QUERY_FIELDS.join(
```

Comment:
Should we make `isoformOverrideSource` a prop with the default value of `mskcc`?